Makefile

```diff
@@ -173,3 +173,49 @@ server-profile-cpu:
 jaeger:
 	open http://localhost:16686
 	docker run --rm -p 5775:5775/udp -p 6831:6831/udp -p 6832:6832/udp -p 5778:5778 -p 16686:16686 -p 14268:14268 -p 9411:9411 -e COLLECTOR_ZIPKIN_HTTP_PORT=9411 jaegertracing/all-in-one:1.7
+
+e2e-setup:
+	make e2e-setup-git
+	make e2e-setup-kind
+	make e2e-setup-fluxd
+
+	@echo "\nSetup complete\n\nRun the following to continue:\n\
+	- make e2e-run-local-daemon\n\
+	- make e2e-run-local-manager\n\
+	- make e2e-do-release"
+
+e2e-teardown:
+	kind delete cluster
+
+e2e-setup-git:
+	rm -rf e2e-test/sourcegitrepo
+	mkdir -p e2e-test/sourcegitrepo/local/releases/default
+	echo "Hello World" > e2e-test/sourcegitrepo/README.md
+	cd e2e-test/sourcegitrepo;\
+	git init;\
+	git add .;\
+	git commit -m "Add readme"
+
+e2e-setup-kind:
+	kind create cluster --config e2e-test/kind-cluster.yaml
+
+e2e-setup-fluxd:
+	kubectl apply -f e2e-test/fluxd.yaml
+
+e2e-run-local-daemon:
+	go run ./cmd/daemon start --environment local --release-manager-url http://localhost:10080
+
+e2e-run-local-manager:
+	go run ./cmd/server start --ssh-private-key ~/.ssh/id_rsa --config-repo file:///Users/Emil/Workspace/lunarway/release-manager/e2e-test/sourcegitrepo --http-port 10080
```

Comment:
I guess this shouldn't point to an absolute local Git repo?

Reply:
no it shouldn't 😅 will fix